nmostesting/suites/IS1101Test.py

```diff
@@ -1733,29 +1744,31 @@ def test_04_03(self, test):
             """
             The test requires streaming from a Sender in order to
             verify the state of the Receiver and the associated outputs.
-            """
-            """
-            if IS11_RefecenceSender is not configured fail 4.3 with log indicating
-            that IS11_RefecenceSender is not configured
+            if IS11_REFERENCE_SENDER is not configured fail 4.3 with log indicating
+            that IS11_REFERENCE_SENDER is not configured
             """
             if len(CONFIG.IS11_REFERENCE_SENDER) == 0:
-                return test.FAIL("IS11_RefecenceSender is not configured")
+                return test.DISABLED("Please configure IS11_REFERENCE_SENDER in Config.py")
             if len(self.receivers_outputs) == 0:
                 return test.UNCLEAR("No IS-11 receivers support outputs")
             for receiver_id in self.receivers_outputs:
-                _, response = TestHelper.do_request('GET', self.node_url + "receivers/" + receiver_id)
+                valid, response = TestHelper.do_request('GET', self.node_url + "receivers/" + receiver_id)
+                if not valid:
+                    return test.FAIL("Unexpected response from the Node API: {}".format(response))
                 if response.status_code != 200:
                     return test.FAIL("The Node API request for receiver {} has failed: {}"
                                      .format(receiver_id, response.json()))
                 receiver_format = response.json()["format"]
                 receiver_transport_file = response.json()["transport"]
                 if (receiver_format == "urn:x-nmos:format:video"):
                     """
-                    using $IS11_RefecenceSender GET sender_id from N?/senders/*/ where
+                    using $IS11_REFERENCE_SENDER GET sender_id from N?/senders/*/ where
                     .format is receiver.format and .transport is receiver.transport
                     """
-                    url = self.node_url.replace(PATTERN.search(self.node_url)[0], CONFIG.IS11_REFERENCE_SENDER)
-                    _, response = TestHelper.do_request('GET', url + "senders/")
+                    url = self.node_url.replace(urlparse(self.node_url).hostname, CONFIG.IS11_REFERENCE_SENDER)
```

Comment:
This doesn't really resolve my previous comment. I still don't think it's appropriate to take the URL of the Node-under-test and apply a different host IP address or host name to it. Obvious problem is if there is a port number, but it's just semantically not the right thing to do. Same goes for the Connection API below.
You are restricting the reference sender to serving its Node API and Connection API on the same address as each other, and on the same ports as the Node-under-test.

Reply:
Now using 4 disctinct variable to give the flexibility in the config file to defined distinct path for Node and Connection API, including variable ports.